resources/language/Swedish/strings.po

```diff
@@ -8,7 +8,7 @@ msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2016-01-27 19:53+0200\n"
-"PO-Revision-Date: 2021-03-11 12:31+0100\n"
+"PO-Revision-Date: 2021-05-23 10:56+0200\n"
 "Last-Translator: \n"
```

Comment:
also if you like you can put your name/nickname here.
e.g. `"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"`